specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/examples/SearchServiceCreateSynonymMapEncryptedWIthCMK.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
nit: capital 'I' in the file name (...WIthCmk)

Reply:
(this file doesn't exist in the latest iteration)